website/docs/cli/commands/modules.mdx

```diff
@@ -0,0 +1,72 @@
+---
+page_title: 'Command: modules'
+description: >-
+  The terraform modules command prints source and version data about all declared
+  modules in Terraform configuration.
+---
+
+# Command: modules
+
+The `terraform modules` command provides a holistic view of all Terraform
+modules declared in Terraform configuration for the current working directory.
+This can be useful for auditing or policy purposes on module consumption. The
+output will include a list of each declared module's key, source and version.
+
+-> `terraform modules` requires **Terraform v1.10.0 or later**.
+
+## Usage
+
+Usage: `terraform modules [options]`
```

Comment:
On second look, I think leaving the json flag in makes sense right now, but I love leaving the list below for when we do add more options. 
```suggestion
Usage: `terraform modules -json`
```